addons/website_slides/views/slide_channel_views.xml

```diff
@@ -12,6 +12,17 @@
                     </header>
                     <sheet>
                         <div class="oe_button_box" name="button_box">
+                            <button name="action_redirect_to_contents"
```

Comment:
As you are on a form view you may re-use action_view_slides. You always have one channel.